components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/listener/AbstractIdentityProviderMgtListener.java

```diff
@@ -61,6 +61,18 @@ public boolean doPreDeleteIdP(String idPName, String tenantDomain) throws Identi
         return true;
     }
 
+    /**
+     * Additional actions before deleting all IdPs of a given tenant id.
+     *
+     * @param tenantDomain Tenant domain to delete IdPs.
```

Comment:
In other places for params full stop is not used. It is better if we can maintain single format for comments. WDYT?